packages/web-components/src/components/tile/tile.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html, property, state, query } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map.js';
```

Comment:
Any directive imports also need updating to [use the `lit` package instead](https://lit.dev/docs/v2/releases/upgrade/#update-directive-imports).